api/health/service.md

```diff
@@ -1,64 +1,38 @@
-
-:::
+<Callout title="Note">
+This API set is for a specific node; it is unavailable on the [public server](/tooling/rpc-providers).
+</Callout>
 
 ## Health Checks
 
 The node periodically runs all health checks, including health checks for each chain.
 
-The frequency at which health checks are run can be specified with the [--health-check-frequency](/nodes/configure/avalanchego-config-flags.md) flag.
+The frequency at which health checks are run can be specified with the [\--health-check-frequency](/nodes/configure/configs-flags) flag.
 
 ## Filterable Health Checks
 
-The health checks that are run by the node are filterable. You can specify which health checks
-you want to see by using `tags` filters. Returned results will only include health checks that
-match the specified tags and global health checks like `network`, `database` etc.
-When filtered, the returned results will not show the full node health,
-but only a subset of filtered health checks.
-This means the node can still be unhealthy in unfiltered checks, even if the returned results show that the node is healthy.
-AvalancheGo supports using subnetIDs as tags.
+The health checks that are run by the node are filterable. You can specify which health checks you want to see by using `tags` filters. Returned results will only include health checks that match the specified tags and global health checks like `network`, `database` etc. When filtered, the returned results will not show the full node health, but only a subset of filtered health checks. This means the node can still be unhealthy in unfiltered checks, even if the returned results show that the node is healthy. AvalancheGo supports using subnetIDs as tags.
 
 ## GET Request
 
-To get an HTTP status code response that indicates the node’s health, make a `GET` request.
-If the node is healthy, it will return a `200` status code.
-If the node is unhealthy, it will return a `503` status code.
-In-depth information about the node's health is included in the response body.
+To get an HTTP status code response that indicates the node's health, make a `GET` request. If the node is healthy, it will return a `200` status code. If the node is unhealthy, it will return a `503` status code. In-depth information about the node's health is included in the response body.
 
 ### Filtering
 
-To filter GET health checks, add a `tag` query parameter to the request.
-The `tag` parameter is a string.
-For example, to filter health results by subnetID `29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL`,
-use the following query:
+To filter GET health checks, add a `tag` query parameter to the request. The `tag` parameter is a string. For example, to filter health results by subnetID `29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL`, use the following query:
 
-```sh
+```
 curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL'
 ```
```

Comment:
why was this formatting removed from every code block? 

```
curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL'
```
```sh
curl 'http://localhost:9650/ext/health?tag=29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL'
```

Reply:
added back. not an issue as the default formatting is picked as `bash` in new docs.